src/dmd/sideeffect.d

```diff
@@ -364,13 +365,11 @@ extern (C++) bool discardValue(Expression e)
  */
 VarDeclaration copyToTemp(StorageClass stc, const char* name, Expression e)
 {
-    assert(name && name[0] == '_' && name[1] == '_');
-    auto id = Identifier.generateId(name);
-    auto ez = new ExpInitializer(e.loc, e);
-    auto vd = new VarDeclaration(e.loc, e.type, id, ez);
-    vd.storage_class = stc;
-    vd.storage_class |= STC.temp;
-    vd.storage_class |= STC.ctfe; // temporary is always CTFEable
+    assert(name[0] == '_' && name[1] == '_');
```

Comment:
Let's hope the caller never passes null.

Reply:
If it is, there'll be a seg fault.